go.mod

```diff
@@ -10,8 +10,8 @@ require (
 	github.com/libp2p/go-libp2p-connmgr v0.2.4
 	github.com/libp2p/go-libp2p-core v0.8.6
 	github.com/libp2p/go-libp2p-discovery v0.5.1
-	github.com/libp2p/go-libp2p-swarm v0.5.2
+	github.com/libp2p/go-libp2p-swarm v0.5.3
```

Comment:
is that intentional?

Reply:
Yes. I needed to release new transports and a new swarm.